spec/swagger.yaml

```diff
@@ -5569,6 +5598,64 @@ definitions:
         },
         "included": []
       }
+  UpsertPatientRequest:
+    type: object
+    required:
+      - meta
+      - data
+    properties:
+      meta:
+        type: object
+        required:
+          - query
```

Comment:
Should 'match' in meta be documented as well along with an example: https://github.com/TwineHealth/TwineServer/blob/master/app/apis/pub/lib/resource.js#L1882

Reply:
I can't find an example of the "match" parameter being used in the tests. Based on the usage in resource.js, I think `body.meta.match` is redundant with `body.meta.query`.